### PR TITLE
Support additional builtin functions

### PR DESCRIPTION
**Pull request by mantognini**
This completes Arm's prototype (#613) with support for some additional builtin functions. Some builtins are still missing and can be added later by any contributor.

### CODE REVIEW

**Comment by polsenferreira** [comment]
@googlebot I fixed it.

**Comment by polsenferreira** [comment]
@googlebot I consent.

**Comment by mantognini** [comment]
@googlebot I consent.

File: lib/LongVectorLoweringPass.cpp [hunk 1]
```diff
@@ -693,6 +736,16 @@ Value *LongVectorLoweringPass::visitCastInst(CastInst &I) {
 
   Value *V = nullptr;
   switch (I.getOpcode()) {
+  case Instruction::BitCast: {
```
[the remark this answers is not in the document]
Comment (mantognini) [reply lib/LongVectorLoweringPass.cpp:739]: You're correct, some casts are not yet supported. I've added a testcase and updated the TODOs in that area.

**Review by alan-baker — APPROVED** [review]

**Pull request merged**